source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -635,7 +635,7 @@ bool COpenGLDriver::initDriver(CIrrDeviceWin32* device)
 #endif // _IRR_COMPILE_WITH_OPENCL_
 	genericDriverInit(device->getAssetManager());
 
-	COpenGLFunctionTable::extGlSwapInterval(Params.Vsync ? 1 : 0);
+    getThreadContext_helper(true)->functionTable.extGlSwapInterval(Params.Vsync ? 1 : 0);
```

Comment:
did you already lock the context mutex? because thats the only time you should call that function with `true`